pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -188,6 +188,7 @@ export class Modifications extends React.Component {
 }
 
 Modifications.propTypes = {
+    failed: PropTypes.bool.isRequired,
```

Comment:
It's not required, it has a fallback.

Reply:
That's true, but it's also something which requires refactoring:

```
    fail_message = failed ? _("Error running semanage to discover system modifications") : fail_message;
```
This makes it SELinux specific

Reply:
Good point -- can you move that message into the selinux page, and keep the required property then?